sys/shell/democommands/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+#![no_std]
+
+use riot_wrappers::println;
+use core::fmt::Write;
+
+riot_wrappers::static_command!(static_hello_world, "hello-world", "Print a greeting", hello_world);
```

Comment:
I think the C macro doesn't allow hyphens, or specifying both a function name and the command name. I don't mind, but maybe, for consistency, rename this to "hello_world"?

Reply:
Sure, adjusted.

The Rust macro tries to be less auto-magic than the C macro (we could stringify, but I chose not to) because that's a kind of automatism I don't think is idiomatic in Rust, but that shouldn't stop from being consistent with the C versions in examples..